ocfweb/templates/base.html

```diff
@@ -100,24 +100,11 @@
         <div class="ocf-status-bar">
             <div class="container">
                 <p>
-                    {% if not lab_status.force_lab_closed %}
-                        {% if lab_is_open %}
-                            <strong>Lab Currently Open</strong>
-                        {% else %}
-                            <strong>Lab Currently Closed</strong>
-                        {% endif %}
-                        <span class="nowrap">
-                            {% if not lab_is_open and current_lab_hours %}
-                                Hours:
-                            {% endif %}
-                            {{current_lab_hours|lab_hours_time}}
-                            on {% now "l" %} {% lab_hours_holiday holidays %}
-                            <a class="subtle" href="{% url 'doc' 'services/lab' %}">more &raquo;</a>
-                        </span>
-                    {% else %}
-                        <strong>Lab Temporarily Closed</strong>
-                        <a class="subtle" href="{% url 'home' %}">more &raquo;</a>
-                    {% endif %}
+                    <strong>Defund the police.</strong>
```

Comment:
I would focus on the movement and steer away from potentially political statements.

Reply:
My personal opinions notwithstanding

Reply:
The movement is political tho, and a common criticism of these kinds of messages is that they want change but aren't clear about what change. "Defund the police" is a bold statement, but one that is apt given current events, clear, and actionable. "Black lives matter", while a more common slogan for tech websites to put up, is much more vague and uncontroversial by virtue of being vague.

Reply:
If you are speaking on behalf of an organization, you should make sure everyone is on board with the message you are presenting

Reply:
While I disagree with others' statements about avoiding politics... We live in a society where the topic of BLM tends is unfortunately inherently political... I think instead my issue with it is that while I would happily approve that it could go into the body I see this like a paper in the sense that this header should be like a topic sentence or title and encapsulate everything we want to say.